### PR TITLE
Added pom to build and updated license to open source license header

### PR DESCRIPTION
**Pull request by dpwspoon**
This license header is required for the pom, I can turn it off if you prefer but it is the one we use in all the other licenses

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <version>develop-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <url>https://github.com/kaazing/universal-client</url>
+    <description>Universal Client</description>
+
+    <scm>
+        <url>https://github.com/kaazing/gateway</url>
+        <connection>scm:git:https://github.com/kaazing/gateway.git</connection>
+    </scm>
```
Comment (ahousing) [inline java/pom.xml]: Wrong location

File: java/pom.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
```
Comment (ahousing) [inline java/pom.xml]: Do we want some sort of comment in this file and build.gradle saying that they need to be kept in sync with each other?

Comment (dpwspoon) [inline java/pom.xml]: That is a question for @nemigaservices ?

**Comment by ahousing** [comment]
It failed to build for me, with errors like:
```
testObject(com.kaazing.client.universal.JMSUniversalClientTest): Error connecting to gateway with ws://localhost:8001/jms, credentials /
```
I tried on three different machines.

**Pull request merged**